airmozilla/main/templates/main/event.html

```diff
@@ -334,6 +335,24 @@ <h1 class="entry-title">
         </div>
       {% endif %}
 
+      {% if event.is_public() %}
+        <div class="tab">
+          <input type="radio" id="tab-share" name="tab-group-1">
+          <label for="tab-share">Share</label>
+          <div class="content">
+            <div>
```

Comment:
I mean this
![captura de pantalla 2015-10-22 a las 16 34 35](https://cloud.githubusercontent.com/assets/363698/10678241/dd6f1b3a-78da-11e5-9faf-e5da3915ed9e.png)